<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> Opciones</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Servidor SMTP</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="servidorLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Puerto</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="puertoSpinBox">
          <property name="maximum">
           <number>69536</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> Seguridad y autenticación</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Seguridad de la conexión</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="seguridadComboBox">
          <item>
           <property name="text">
            <string>Ninguna</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SSL/TLS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>STARTTLS</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Metodo de autenticación</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="autenticacionComboBox">
          <item>
           <property name="text">
            <string>Sin autenticación</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Contraseña normal</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_16">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string> Usuario</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ususario</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usuarioLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Contaseña</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="claveLineEdit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="3" column="2">
        <widget class="QToolButton" name="quitarToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/img/remove.png</normaloff>:/img/remove.png</iconset>
         </property>
        </widget>
       </item>
       <item row="4" column="2" rowspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Cuerpo</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="4">
        <widget class="QListWidget" name="adjuntosListWidget">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QTextEdit" name="cuerpoTextEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Adjunto</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="agregarToolButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resource.qrc">
           <normaloff>:/img/add.png</normaloff>:/img/add.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="enviarPushButton">
        <property name="text">
         <string>Enviar Correo</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/mail-send.png</normaloff>:/img/mail-send.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="closePushButton">
        <property name="text">
         <string>Cerrar</string>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/img/log-out.png</normaloff>:/img/log-out.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="verticalSpacing">
        <number>1</number>
       </property>
       <item row="3" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Para:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Asunto</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="deNombreLineEdit">
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>De:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>E-Mail</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="deCorreoLineEdit"/>
       </item>
       <item row="3" column="1" colspan="4">
        <widget class="QLineEdit" name="paraLineEdit"/>
       </item>
       <item row="4" column="1" colspan="4">
        <widget class="QLineEdit" name="asuntoLineEdit"/>
       </item>
       <item row="2" column="0" colspan="5">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>closePushButton</tabstop>
  <tabstop>servidorLineEdit</tabstop>
  <tabstop>puertoSpinBox</tabstop>
  <tabstop>seguridadComboBox</tabstop>
  <tabstop>autenticacionComboBox</tabstop>
  <tabstop>usuarioLineEdit</tabstop>
  <tabstop>claveLineEdit</tabstop>
  <tabstop>deNombreLineEdit</tabstop>
  <tabstop>paraLineEdit</tabstop>
  <tabstop>asuntoLineEdit</tabstop>
  <tabstop>cuerpoTextEdit</tabstop>
  <tabstop>adjuntosListWidget</tabstop>
  <tabstop>agregarToolButton</tabstop>
  <tabstop>quitarToolButton</tabstop>
  <tabstop>enviarPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
